src/io/flutter/settings/FlutterSettings.java

```diff
@@ -31,6 +35,8 @@
   private static final String disableTrackWidgetCreationKey = "io.flutter.disableTrackWidgetCreation";
   private static final String useFlutterLogView = "io.flutter.useLogView";
   private static final String memoryProfilerKey = "io.flutter.memoryProfiler";
+  private static final String newBazelTestRunnerKey = "io.flutter.bazel.legacyTestBehavior";
+  private static final boolean newBazelTestRunnerDefaultValue = false;
```

Comment:
I would just in-line this value into the two use sites (like we do elsewhere in the file).

Reply:
Done.